validator/client/propose.go

```diff
@@ -85,8 +80,29 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64, pubKey [48]by
 		Signature: sig,
 	}
 
-	if err := v.postBlockSignUpdate(ctx, pubKey, blk, domain); err != nil {
-		log.WithField("slot", blk.Block.Slot).WithError(err).Error("Failed post block signing validations")
+	slashable, err := v.localSlashingProtector.IsSlashableBlock(ctx, blk, pubKey, signingRoot)
+	if err != nil {
+		log.WithFields(
+			blockLogFields(pubKey, blk),
+		).WithError(err).Error("Could not check block safety with slashing protection, not submitting")
+		return
+	}
+	if remoteProtector, ok := v.remoteSlashingProtector.(*remote.Service); ok && remoteProtector != nil {
+		slashable, err = v.remoteSlashingProtector.IsSlashableBlock(ctx, blk, pubKey, signingRoot)
+		if err != nil {
+			// If slasher is unavailable, trust local protection and proceed with submitting the attestation.
+			if !errors.Is(err, remote.ErrSlasherUnavailable) {
+				log.WithFields(
+					blockLogFields(pubKey, blk),
+				).WithError(err).Error("Could not check block safety with slashing protection, not submitting")
+				return
+			}
```

Comment:
Same here. Lets log a warning if the slasher is unavailable to process our request.